frontend/vendor/js/diff.min.js

```diff
@@ -0,0 +1,307 @@
+/*!
+
+ diff v3.1.0
```

Comment:
I was so convinced lineman would pick up libs from `node_modules`, that I just tried it out myself to find out it is not working and thought we had something wrong with out lineman config. 

Later found out lineman encourages bower-style of adding libs to `vendor`. Sigh.. 😞  http://linemanjs.com/#fetching-3rd-party-libraries